### ISSUE TITLE
rector.phar: [ERROR] Class '_HumbugBox0997c782eec9\Symplify\EasyCodingStandard\Configuration\Option' not found 

### ISSUE THREAD

**Issue by lptn** [rectorphp/rector#3721]
# Bug Report

![image](https://user-images.githubusercontent.com/5278175/87651823-16b37600-c75c-11ea-8c56-cd6f475cdc85.png)

`rector.phar` downloaded from: https://github.com/rectorphp/rector-prefixed/raw/master/rector.phar

**Comment by TomasVotruba** [comment]
Hi,

what is your minimal `rector.yaml` configuration that causes this?

**Comment by lptn** [comment]
@TomasVotruba 
it fails with any config. example:
```yml
services:
    Rector\CodeQuality\Rector\Ternary\ArrayKeyExistsTernaryThenValueToCoalescingRector: ~

parameters:
    paths:
        - 'app'
    php_version_features: '7.4'
```
error (new build throws a new exception):
```
                                                                                                                 
 [ERROR] Expected to find class "Symplify\ConsoleColorDiff\Console\Formatter\ColorConsoleDiffFormatter" in file         
         "phar:///Users/username/code/rector.phar/vendor/symplify/console-color-diff/config/../src
         /Console/Formatter/ColorConsoleDiffFormatter.php" while importing services from resource                       
         "phar:///Users/username/code/rector.phar/vendor/symplify/console-color-diff/config/../src
         ", but it was not found! Check the namespace prefix used with the resource.                                    
                                                                                                    
```

even --version:

![image](https://user-images.githubusercontent.com/5278175/87849582-675cd780-c8f2-11ea-9cda-1cdd74b7638f.png)

**Comment by TomasVotruba** [comment]
Should be resolved in 0.7.53
